.travis/install_terraform.sh

```diff
@@ -3,7 +3,6 @@
 set -e
 
 wget https://releases.hashicorp.com/terraform/0.11.10/terraform_0.11.10_linux_amd64.zip
-unzip terraform_0.11.10_linux_amd64.zip
-sudo mv terraform /usr/local/bin/
+sudo unzip -o terraform_0.11.10_linux_amd64.zip -d /usr/local/bin/
```

Comment:
We use `-d /usr/local/bin/` to unzip to `/usr/local/bin/`. We use `-o` to overwrite any existing file. I initially added `-o` when we were unzipping into the repository root, resulting in a conflict with the existing `terraform/` directory.